hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -416,7 +416,7 @@ def reset_add_line(self):
             sheet.add_line_project_id = False
 
     def _get_date_name(self, date):
-        return date.strftime("%a\n%b %d")
+        return date.strftime(_('%a,\n%b&nbsp;%d'))
```

Comment:
Why put this as translatable? You should pass a translatable string only the part that is really translatable, or there are chances that this breaks the system in a failure translation.

Reply:
Because `Mon, Feb 15` is locale specific

Reply:
I wish it could've been a way to get similar format automatically yet I haven't found a way

Reply:
But for that you should rely on `strftime` itself with proper language. The string to be translated with this is going to be `%a,\n%b&nbsp;%d`, not `Mon, Feb 15`.

Reply:
(language or locale)

Reply:
Exactly what I'm after, e.g. for Ukrainian it would have to be `%a,\n%d&nbsp;%b` (day-month vs month-day)

Reply:
But this is already put on each language. You have to use the information in res.lang for this.

Reply:
Not sure I'm following you

Reply:
![seleccion_021](https://user-images.githubusercontent.com/7165771/52872471-df96fa80-314c-11e9-9f9b-80e70a303b7b.png)

I know it's not the same format but:

A) You can add a module for short date/long date format added to res.lang, and use that one.
B) Or translate numerical format to short format, as it follows the same scheme.

Reply:
I've checked all known datetime-related libraries (from other languages as well), no other maintains a valid database of such things

A) seems to be an overkill - I haven't seen other usages so far
B) well, it would be impossible to test. I've thought of this like "format as usual and remove year from string using replace"

Both seem not ok. I've seen translation files with html and other tech-specific stuff, I think it's not _that_ a big risk after all

Reply:
Yeah, there are these codes, and you can see issues coming from this. They are working for removing all these non translatable stuff from master for avoiding problems (switching HTML or Python code). Let me search alternatives.

Reply:
http://userguide.icu-project.org/formatparse/datetime would help, mentioned here https://github.com/odoo/odoo/blob/728bb2a28ec84514e3fd63a3732822ed34c028a8/odoo/tools/misc.py#L644 and https://github.com/odoo/odoo/blob/728bb2a28ec84514e3fd63a3732822ed34c028a8/odoo/tools/misc.py#L1166

Reply:
Good to know that this http://babel.pocoo.org/en/latest/api/dates.html is available in Odoo

Reply:
This one is useful http://babel.pocoo.org/en/latest/api/core.html#babel.core.Locale.datetime_skeletons